### PR TITLE
MEED-395 Add market values into Meeds Token section

### PR DESCRIPTION
**Pull request by MayTekayaa**
Before this change, the Meed tokens section contains only the Meed price as details.
This change will add other metrics such as market capitalization, circulating supply and total locked value.

### CODE REVIEW

File: deeds-dapp-webapp/src/main/webapp/vue-app/snapshot/components/MeedsInfo.vue [hunk 1]
```diff
@@ -24,39 +24,88 @@
...
+          {{ totalValuelocked }}
+        </v-list-item-content>
       </v-list-item>
     </v-list>
     <div class="d-flex flex-column flex-sm-row">
       <deeds-price-chart class="mb-4 mb-sm-8" />
-      <deeds-currencies-chart class="mt-15 ml-8" />
+      <deeds-currencies-chart 
+        class="mt-15 ml-8"
+        @metrics="getMetrics" />
```
Comment (boubaker) [inline deeds-dapp-webapp/src/main/webapp/vue-app/snapshot/components/MeedsInfo.vue]:
```suggestion
        @metrics="setMetrics" />
```

File: deeds-dapp-webapp/src/main/webapp/vue-app/snapshot/components/charts/CurrenciesChart.vue [hunk 2]
```diff
@@ -109,6 +109,7 @@ export default {
     this.$tokenMetricService.getMetrics()
```
Comment (boubaker) [inline deeds-dapp-webapp/src/main/webapp/vue-app/snapshot/components/charts/CurrenciesChart.vue]: it would be better to move this to parent

Comment (MayTekayaa) [reply deeds-dapp-webapp/src/main/webapp/vue-app/snapshot/components/charts/CurrenciesChart.vue]: done

File: deeds-dapp-webapp/src/main/webapp/vue-app/snapshot/components/MeedsInfo.vue (hunk 1 above)
Comment (MayTekayaa) [reply deeds-dapp-webapp/src/main/webapp/vue-app/snapshot/components/MeedsInfo.vue]: done

**Review by boubaker — APPROVED** [review]

**Pull request merged**